packages/gasket-core/lib/index.d.ts

```diff
@@ -50,6 +50,7 @@ declare module '@gasket/core' {
 
   // This is the config
   export interface GasketConfig {
+    filename?: string;
```

Comment:
So you're declaring the name of the file in which this is defined? In `gasket.js` you'll have a `filename` property with the value of `gasket.js`?

Reply:
Basically yes, though it'll be the full path i.e. `/absolute/path/to/gasket.js`. By using `import.meta.filename` we don't have to worry if the user renames or moves it.